app/components/AuthEntry.js

```diff
@@ -69,24 +81,41 @@ export default class AuthEntry extends React.Component {
     this.props.onCopyValue();
   }
 
-  render() {
-    const { service, account, notes, color, password } = this.props.entry;
-    const { id, onEdit, onRemove, canEdit, style, innerRef, ...divProps } = this.props;
-    const { token, timeLeft } = this.state;
+  updateEntryStyle = (isOnMount = false) => {
```

Comment:
```suggestion
  updateEntryStyle = (useDelay = true) => {
```

(don't commit suggestion directly, just proposing)